pkgpanda/build/__init__.py

```diff
@@ -220,6 +220,8 @@ def package_tuples_with_dependencies(package_names, treeinfo, package_store):
     @staticmethod
     def validate_package_tuples(package_tuples, treeinfo, package_store):
         # Validate that all packages have the variant specified in treeinfo.
+        print('package_tuples = %r' % package_tuples)
```

Comment:
Are these lines debug output that shouldn't be printed in final build?

Reply:
These lines provide the information that lets us better identify the problem in the future. `print` is currently `pkgpanda`'s log method, at least on the build side.